app/src/main/java/org/httpsknighthacks/knighthacksandroid/HorizontalSectionCard_RecyclerViewAdapter.java

```diff
@@ -30,6 +36,10 @@
     private ArrayList<String> mCardBodyList;
     private ArrayList<String> mCardTimestampList;
     private ArrayList<String> mCardFooterList;
+    private ArrayList<String> mCardMapEventList;
+    private String tag;
+
+    Dialog mDialog;
```

Comment:
I'm not sure if Dialog should be a member variable of this RecyclerView since it's only needed for Workshops and ScheduleEvents. Can you make it a local variable?